src/test/java/net/snowflake/client/jdbc/CustomProxyLatestIT.java

```diff
@@ -240,31 +249,33 @@ public void testSizeOfHttpClientNoProxies() throws SQLException {
     props.put("user", "USER");
     props.put("password", "PASSWORD");
     // Set up the first connection and proxy
-    Connection con1 =
-        DriverManager.getConnection(
-            "jdbc:snowflake://s3testaccount.us-east-1.snowflakecomputing.com", props);
-    Statement stmt = con1.createStatement();
-    ResultSet rs = stmt.executeQuery("select 1");
-    rs.next();
-    assertEquals(1, rs.getInt(1));
+    try (Connection con1 =
+            DriverManager.getConnection(
+                "jdbc:snowflake://s3testaccount.us-east-1.snowflakecomputing.com", props);
+        Statement stmt = con1.createStatement();
+        ResultSet rs = stmt.executeQuery("select 1")) {
+      rs.next();
+      assertEquals(1, rs.getInt(1));
+    }
+
     // put in some fake properties that won't get picked up because useProxy=false
     props.put("useProxy", false);
     props.put("proxyHost", "localhost");
     props.put("proxyPort", "8080");
-    Connection con2 =
+    try (Connection con2 =
```

Comment:
we can rename all con1, con2, con3 with simple `con` when the connection has small scope

Reply:
replaced.